print_deps.rb

```diff
@@ -158,13 +148,25 @@ def print_yaml(output_file, packages)
   File.write(output_file, result_map.to_yaml)
 end
 
-project, target, arch, output_file = ARGV
+project, target, arch, output_dir = ARGV
+
+raise "Usage: $0 <obs_project> <obs_build_target> <arch> <output_dir>" unless output_dir
+Requires.check_environment
 
-raise "Usage: $0 <obs_project> <obs_build_target> <arch> <output_file>" unless output_file
+# build requires
+build_packages = get_packages(project, target, arch)
+compute_layers(build_packages)
+compute_time(build_packages, project, target, arch)
+compute_min_time(build_packages)
+print_yaml(File.join(output_dir,"build_requires.yaml"), build_packages)
 
-packages = get_packages(project, target, arch)
+# installation requires
+repositories = { "openSUSE_Factory" => "obs://openSUSE:Factory/standard",
+  "openSUSE_42.1" => "obs://leap/42.1/repo/oss",
+  "openSUSE_Leap_42.2" => "obs://leap/42.1/repo/oss" }
```

Comment:
please document this script help, what repositories it allows like ` supported repositories is #{repositories.keys.join(",")`